src/contrib/tf_op/index_seq.h

```diff
@@ -0,0 +1,44 @@
+/**
+ * Refer to std::index_sequence (since c++14)
+ * Utilities to invoke variadic function with template <size_t N> 
+ */
+#ifndef __TFTVM_INDEX_SEQ__
+#define __TFTVM_INDEX_SEQ__
+
+template <std::size_t ...>
```

Comment:
Do we need these helpers? I think we can probably directly invoke using PackedFunc, right?

Reply:
> Do we need these helpers? I think we can probably directly invoke using PackedFunc, right?

PackedFunc is invoked variadic but DLPack input tensors is array. Thus direct implementation maybe like:
```c++
switch(NUM_INPUTS) {
case 1 :
  f(args[0]); break;
case 2:
  f(args[0], args[1]); break;
case 3:
  f(args[0], args[1], args[2]); break;
// update to 8 ...
}
```
This seems quite strange so we use some template utility to call the PackedFunc.